### PR TITLE
support the title slot in menu-group

### PR DESCRIPTION
**Pull request by baiyaaaaa**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by baiyaaaaa** [comment]
#1334

File: examples/docs/en-US/menu.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Top bar NavMenu can be used in a variety of scenarios.
 
 Vertical NavMenu with sub-menus.
 
-::: demo
+::: demo you can use `el-menu-item-group` component to create the menu group，the name of the group is setting by the `title` prop or the name slot.
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/menu.md]: You can use the `el-menu-item-group` component to create a menu group, and the name of the group is determined by the `title` prop or a named slot.

**Comment by baiyaaaaa** [comment]
done

**Comment by shuhh9527** [comment]
敢问动态获取二级菜单后，二级菜单的定位出现问题，笔者有遇到过么？

**Comment by shuhh9527** [comment]
敢问动态获取二级菜单后，二级菜单的定位出现问题且不显示，笔者有遇到过么？

**Pull request merged**